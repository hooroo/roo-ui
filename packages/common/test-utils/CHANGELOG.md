# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

<a name="0.4.0"></a>
# [0.4.0](https://github.com/hooroo/roo-ui/compare/v0.3.3...v0.4.0) (2018-04-26)


### Features

* add theming ([bee896a](https://github.com/hooroo/roo-ui/commit/bee896a))
