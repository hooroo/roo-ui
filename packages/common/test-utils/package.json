{
  "name": "@roo-ui/test-utils",
  "version": "0.0.0",
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "build": "babel src -d dist --copy-files"
  }
}
