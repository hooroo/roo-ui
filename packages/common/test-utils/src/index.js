export { default as shallowWithTheme } from './shallowWithTheme'; // eslint-disable-line import/prefer-default-export
