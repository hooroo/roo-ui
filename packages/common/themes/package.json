{
  "name": "@roo-ui/themes",
  "version": "0.4.0",
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "build": "babel src -d dist --copy-files"
  }
}
