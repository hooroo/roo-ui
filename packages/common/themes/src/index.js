import qantas from './qantas';

export { qantas }; // eslint-disable-line import/prefer-default-export
