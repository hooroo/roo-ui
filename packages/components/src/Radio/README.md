# Radio

<!-- STORY -->

## Installation

```shell
$ yarn add @roo-ui/components
```

## Example

```js
import { Box, Label, Radio } from '@roo-ui/components';

export default (
  <Box textAlign="left" is="form">
    <Label textStyle="text">
      <Radio name="example" /> Uno
    </Label>

    <Label textStyle="text">
      <Radio name="example" /> Dos
    </Label>

    <Label textStyle="text">
      <Radio name="example" /> Tres
    </Label>

    <Label textStyle="text">
      <Radio disabled name="example" /> Quattro
    </Label>
  </Box>
);
```

## Properties

| Name       | Description           | Type   | Default | Required? |
|:-----------|:----------------------|:-------|:--------|:----------|
| `checked`  | sets checked state    | `bool` | false   | -         |
| `disabled` | disables radio button | `bool` | false   | -         |

Standard [radio attributes](https://www.w3schools.com/jsref/dom_obj_radio.asp) will be passed down to the `<input />`.
