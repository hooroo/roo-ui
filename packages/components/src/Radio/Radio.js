import React from 'react';
import styled from 'styled-components';
import { themeGet } from 'styled-system';
import { rem } from 'polished';
import tag from 'clean-tag';

const Radio = props => (
  <RadioWrapper>
    <RadioInput type="radio" {...props} />
    <RadioIcon />
  </RadioWrapper>
);

const RadioWrapper = styled(tag.div)`
  display: inline-block;
  position: relative;
`;

const RadioInput = styled.input`
  appearance: none;
  opacity: 0;
  position: absolute;
  z-index: 0;

  &:focus {
    box-shadow: none;
  }
`;

const RadioIcon = styled(tag.div)`
  display: inline-block;
  position: relative;
  vertical-align: middle;
  margin-bottom: ${rem('2px')};
  margin-right: ${themeGet('space.2')};
  height: ${rem('20px')};
  width: ${rem('20px')};
  background-color: ${themeGet('colors.white')};
  border: ${themeGet('borders.2')};
  border-color: ${themeGet('colors.grey.2')};
  border-radius: 50%;
  z-index: 1;

  input:checked + &,
  input:focus + &,
  input:hover + & {
    border-color: ${themeGet('colors.brand.secondary')};
    transition: ${themeGet('transitions.default')}
  }

  input:checked + & {
    &:after {
      position: absolute;
      display: block;
      content: '';
      top: ${themeGet('space.1')};
      left: ${themeGet('space.1')};
      height: ${themeGet('space.2')};
      width: ${themeGet('space.2')};
      border-radius: 50%;
      background-color: black;
    }
  }

  input:disabled + & {
    background-color: ${themeGet('colors.grey.2')};
    border-color: ${themeGet('colors.grey.2')};
  }
`;

export default Radio;
