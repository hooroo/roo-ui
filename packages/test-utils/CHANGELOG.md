# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

<a name="0.50.7"></a>
## [0.50.7](https://github.com/hooroo/roo-ui/compare/v0.50.6...v0.50.7) (2018-08-14)


### Bug Fixes

* pass theme to child components; update snapshots ([#153](https://github.com/hooroo/roo-ui/issues/153)) ([1a83477](https://github.com/hooroo/roo-ui/commit/1a83477))




<a name="0.39.1"></a>
## [0.39.1](https://github.com/hooroo/roo-ui/compare/v0.39.0...v0.39.1) (2018-06-26)




**Note:** Version bump only for package @roo-ui/test-utils

<a name="0.39.0"></a>
# [0.39.0](https://github.com/hooroo/roo-ui/compare/v0.38.4...v0.39.0) (2018-06-26)


### Features

* add build:watch scripts ([#98](https://github.com/hooroo/roo-ui/issues/98)) ([bcae0bf](https://github.com/hooroo/roo-ui/commit/bcae0bf))




<a name="0.32.0"></a>
# [0.32.0](https://github.com/hooroo/roo-ui/compare/v0.31.0...v0.32.0) (2018-06-15)


### Features

* add autocomplete component ([#77](https://github.com/hooroo/roo-ui/issues/77)) ([eee3f8c](https://github.com/hooroo/roo-ui/commit/eee3f8c))




<a name="0.19.0"></a>
# [0.19.0](https://github.com/hooroo/roo-ui/compare/v0.18.0...v0.19.0) (2018-05-22)


### Features

* Add <List/> & <ListItem> ([#49](https://github.com/hooroo/roo-ui/issues/49)) ([b922a8a](https://github.com/hooroo/roo-ui/commit/b922a8a))




<a name="0.7.0"></a>
# [0.7.0](https://github.com/hooroo/roo-ui/compare/v0.6.4...v0.7.0) (2018-05-11)


### Features

* add new icon set ([#14](https://github.com/hooroo/roo-ui/issues/14)) ([377c4c8](https://github.com/hooroo/roo-ui/commit/377c4c8))




<a name="0.5.4"></a>
## [0.5.4](https://github.com/hooroo/roo-ui/compare/v0.5.2...v0.5.4) (2018-05-08)


### Bug Fixes

* replace build script with prepublish ([90cafe3](https://github.com/hooroo/roo-ui/commit/90cafe3))




<a name="0.5.3"></a>
## [0.5.3](https://github.com/hooroo/roo-ui/compare/v0.5.2...v0.5.3) (2018-05-08)


### Bug Fixes

* replace build script with prepublish ([90cafe3](https://github.com/hooroo/roo-ui/commit/90cafe3))




<a name="0.5.0"></a>
# [0.5.0](https://github.com/hooroo/roo-ui/compare/v0.4.2...v0.5.0) (2018-05-07)


### Features

* add ui primitives ([#13](https://github.com/hooroo/roo-ui/issues/13)) ([5789b96](https://github.com/hooroo/roo-ui/commit/5789b96))




<a name="0.4.2"></a>
## [0.4.2](https://github.com/hooroo/roo-ui/compare/v0.4.0...v0.4.2) (2018-04-26)




**Note:** Version bump only for package @roo-ui/test-utils

<a name="0.4.1"></a>
## [0.4.1](https://github.com/hooroo/roo-ui/compare/v0.4.0...v0.4.1) (2018-04-26)




**Note:** Version bump only for package @roo-ui/test-utils

<a name="0.4.0"></a>
# [0.4.0](https://github.com/hooroo/roo-ui/compare/v0.3.3...v0.4.0) (2018-04-26)


### Features

* add theming ([bee896a](https://github.com/hooroo/roo-ui/commit/bee896a))
