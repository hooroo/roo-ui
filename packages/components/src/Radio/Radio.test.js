import React from 'react';
import { qantas as theme } from '@roo-ui/themes';
import { mountWithTheme } from '@roo-ui/test-utils';
import { axe } from 'jest-axe';

import Radio from '.';

describe('<Radio />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mountWithTheme(
      <Radio />,
      theme,
    );
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has no accessibility errors', async () => {
    wrapper = mountWithTheme(<label htmlFor="radio">Label<Radio /></label>, theme);
    expect(await axe(wrapper.html())).toHaveNoViolations();
  });

  describe('when checked', () => {
    beforeEach(() => {
      wrapper = mountWithTheme(
        <Radio checked />,
        theme,
      );
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('when disabled', () => {
    beforeEach(() => {
      wrapper = mountWithTheme(
        <Radio disabled />,
        theme,
      );
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
