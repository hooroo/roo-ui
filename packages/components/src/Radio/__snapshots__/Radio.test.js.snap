// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> renders correctly 1`] = `
.c0 {
  display: inline-block;
  position: relative;
}

.c1 {
  opacity: 0;
  position: absolute;
  z-index: 0;
}

.c1:focus {
  box-shadow: none;
}

.c2 {
  display: inline-block;
  position: relative;
  vertical-align: middle;
  margin-bottom: 0.125rem;
  height: 1.25rem;
  width: 1.25rem;
  border-radius: 50%;
  z-index: 1;
}

input:checked + .c2:after {
  position: absolute;
  display: block;
  content: '';
  border-radius: 50%;
}

<Radio>
  <Radio__RadioWrapper>
    <Clean.div
      className="c0"
    >
      <Clean.div
        blacklist={
          Array [
            "m",
            "mt",
            "mr",
            "mb",
            "ml",
            "mx",
            "my",
            "p",
            "pt",
            "pr",
            "pb",
            "pl",
            "px",
            "py",
            "width",
            "w",
            "fontSize",
            "f",
            "color",
            "bg",
            "fontFamily",
            "font",
            "textAlign",
            "align",
            "lineHeight",
            "fontWeight",
            "letterSpacing",
            "display",
            "maxWidth",
            "minWidth",
            "height",
            "maxHeight",
            "minHeight",
            "size",
            "ratio",
            "alignItems",
            "alignContent",
            "justifyContent",
            "justify",
            "flexWrap",
            "wrap",
            "flexBasis",
            "flexDirection",
            "flex",
            "justifySelf",
            "alignSelf",
            "order",
            "gridGap",
            "gridColumnGap",
            "gridRowGap",
            "gridColumn",
            "gridRow",
            "gridAutoFlow",
            "gridAutoColumns",
            "gridAutoRows",
            "gridTemplateColumns",
            "gridTemplateRows",
            "border",
            "borderTop",
            "borderRight",
            "borderBottom",
            "borderLeft",
            "borderColor",
            "borderRadius",
            "boxShadow",
            "background",
            "backgroundImage",
            "bgImage",
            "backgroundSize",
            "bgSize",
            "backgroundPosition",
            "bgPosition",
            "backgroundRepeat",
            "bgRepeat",
            "position",
            "zIndex",
            "top",
            "right",
            "bottom",
            "left",
            "hover",
            "focus",
            "active",
            "disabledStyle",
            "textStyle",
            "colors",
            "buttonStyle",
            "borderWidth",
          ]
        }
        className="c0"
        is="div"
      >
        <div
          className="c0"
        >
          <Radio__RadioInput
            type="radio"
          >
            <input
              className="c1"
              type="radio"
            />
          </Radio__RadioInput>
          <Radio__RadioIcon>
            <Clean.div
              className="c2"
            >
              <Clean.div
                blacklist={
                  Array [
                    "m",
                    "mt",
                    "mr",
                    "mb",
                    "ml",
                    "mx",
                    "my",
                    "p",
                    "pt",
                    "pr",
                    "pb",
                    "pl",
                    "px",
                    "py",
                    "width",
                    "w",
                    "fontSize",
                    "f",
                    "color",
                    "bg",
                    "fontFamily",
                    "font",
                    "textAlign",
                    "align",
                    "lineHeight",
                    "fontWeight",
                    "letterSpacing",
                    "display",
                    "maxWidth",
                    "minWidth",
                    "height",
                    "maxHeight",
                    "minHeight",
                    "size",
                    "ratio",
                    "alignItems",
                    "alignContent",
                    "justifyContent",
                    "justify",
                    "flexWrap",
                    "wrap",
                    "flexBasis",
                    "flexDirection",
                    "flex",
                    "justifySelf",
                    "alignSelf",
                    "order",
                    "gridGap",
                    "gridColumnGap",
                    "gridRowGap",
                    "gridColumn",
                    "gridRow",
                    "gridAutoFlow",
                    "gridAutoColumns",
                    "gridAutoRows",
                    "gridTemplateColumns",
                    "gridTemplateRows",
                    "border",
                    "borderTop",
                    "borderRight",
                    "borderBottom",
                    "borderLeft",
                    "borderColor",
                    "borderRadius",
                    "boxShadow",
                    "background",
                    "backgroundImage",
                    "bgImage",
                    "backgroundSize",
                    "bgSize",
                    "backgroundPosition",
                    "bgPosition",
                    "backgroundRepeat",
                    "bgRepeat",
                    "position",
                    "zIndex",
                    "top",
                    "right",
                    "bottom",
                    "left",
                    "hover",
                    "focus",
                    "active",
                    "disabledStyle",
                    "textStyle",
                    "colors",
                    "buttonStyle",
                    "borderWidth",
                  ]
                }
                className="c2"
                is="div"
              >
                <div
                  className="c2"
                />
              </Clean.div>
            </Clean.div>
          </Radio__RadioIcon>
        </div>
      </Clean.div>
    </Clean.div>
  </Radio__RadioWrapper>
</Radio>
`;

exports[`<Radio /> when checked renders correctly 1`] = `
.c0 {
  display: inline-block;
  position: relative;
}

.c1 {
  opacity: 0;
  position: absolute;
  z-index: 0;
}

.c1:focus {
  box-shadow: none;
}

.c2 {
  display: inline-block;
  position: relative;
  vertical-align: middle;
  margin-bottom: 0.125rem;
  height: 1.25rem;
  width: 1.25rem;
  border-radius: 50%;
  z-index: 1;
}

input:checked + .c2:after {
  position: absolute;
  display: block;
  content: '';
  border-radius: 50%;
}

<Radio
  defaultChecked={true}
>
  <Radio__RadioWrapper>
    <Clean.div
      className="c0"
    >
      <Clean.div
        blacklist={
          Array [
            "m",
            "mt",
            "mr",
            "mb",
            "ml",
            "mx",
            "my",
            "p",
            "pt",
            "pr",
            "pb",
            "pl",
            "px",
            "py",
            "width",
            "w",
            "fontSize",
            "f",
            "color",
            "bg",
            "fontFamily",
            "font",
            "textAlign",
            "align",
            "lineHeight",
            "fontWeight",
            "letterSpacing",
            "display",
            "maxWidth",
            "minWidth",
            "height",
            "maxHeight",
            "minHeight",
            "size",
            "ratio",
            "alignItems",
            "alignContent",
            "justifyContent",
            "justify",
            "flexWrap",
            "wrap",
            "flexBasis",
            "flexDirection",
            "flex",
            "justifySelf",
            "alignSelf",
            "order",
            "gridGap",
            "gridColumnGap",
            "gridRowGap",
            "gridColumn",
            "gridRow",
            "gridAutoFlow",
            "gridAutoColumns",
            "gridAutoRows",
            "gridTemplateColumns",
            "gridTemplateRows",
            "border",
            "borderTop",
            "borderRight",
            "borderBottom",
            "borderLeft",
            "borderColor",
            "borderRadius",
            "boxShadow",
            "background",
            "backgroundImage",
            "bgImage",
            "backgroundSize",
            "bgSize",
            "backgroundPosition",
            "bgPosition",
            "backgroundRepeat",
            "bgRepeat",
            "position",
            "zIndex",
            "top",
            "right",
            "bottom",
            "left",
            "hover",
            "focus",
            "active",
            "disabledStyle",
            "textStyle",
            "colors",
            "buttonStyle",
            "borderWidth",
          ]
        }
        className="c0"
        is="div"
      >
        <div
          className="c0"
        >
          <Radio__RadioInput
            defaultChecked={true}
            type="radio"
          >
            <input
              className="c1"
              defaultChecked={true}
              type="radio"
            />
          </Radio__RadioInput>
          <Radio__RadioIcon>
            <Clean.div
              className="c2"
            >
              <Clean.div
                blacklist={
                  Array [
                    "m",
                    "mt",
                    "mr",
                    "mb",
                    "ml",
                    "mx",
                    "my",
                    "p",
                    "pt",
                    "pr",
                    "pb",
                    "pl",
                    "px",
                    "py",
                    "width",
                    "w",
                    "fontSize",
                    "f",
                    "color",
                    "bg",
                    "fontFamily",
                    "font",
                    "textAlign",
                    "align",
                    "lineHeight",
                    "fontWeight",
                    "letterSpacing",
                    "display",
                    "maxWidth",
                    "minWidth",
                    "height",
                    "maxHeight",
                    "minHeight",
                    "size",
                    "ratio",
                    "alignItems",
                    "alignContent",
                    "justifyContent",
                    "justify",
                    "flexWrap",
                    "wrap",
                    "flexBasis",
                    "flexDirection",
                    "flex",
                    "justifySelf",
                    "alignSelf",
                    "order",
                    "gridGap",
                    "gridColumnGap",
                    "gridRowGap",
                    "gridColumn",
                    "gridRow",
                    "gridAutoFlow",
                    "gridAutoColumns",
                    "gridAutoRows",
                    "gridTemplateColumns",
                    "gridTemplateRows",
                    "border",
                    "borderTop",
                    "borderRight",
                    "borderBottom",
                    "borderLeft",
                    "borderColor",
                    "borderRadius",
                    "boxShadow",
                    "background",
                    "backgroundImage",
                    "bgImage",
                    "backgroundSize",
                    "bgSize",
                    "backgroundPosition",
                    "bgPosition",
                    "backgroundRepeat",
                    "bgRepeat",
                    "position",
                    "zIndex",
                    "top",
                    "right",
                    "bottom",
                    "left",
                    "hover",
                    "focus",
                    "active",
                    "disabledStyle",
                    "textStyle",
                    "colors",
                    "buttonStyle",
                    "borderWidth",
                  ]
                }
                className="c2"
                is="div"
              >
                <div
                  className="c2"
                />
              </Clean.div>
            </Clean.div>
          </Radio__RadioIcon>
        </div>
      </Clean.div>
    </Clean.div>
  </Radio__RadioWrapper>
</Radio>
`;

exports[`<Radio /> when disabled renders correctly 1`] = `
.c0 {
  display: inline-block;
  position: relative;
}

.c1 {
  opacity: 0;
  position: absolute;
  z-index: 0;
}

.c1:focus {
  box-shadow: none;
}

.c2 {
  display: inline-block;
  position: relative;
  vertical-align: middle;
  margin-bottom: 0.125rem;
  height: 1.25rem;
  width: 1.25rem;
  border-radius: 50%;
  z-index: 1;
}

input:checked + .c2:after {
  position: absolute;
  display: block;
  content: '';
  border-radius: 50%;
}

<Radio
  disabled={true}
>
  <Radio__RadioWrapper>
    <Clean.div
      className="c0"
    >
      <Clean.div
        blacklist={
          Array [
            "m",
            "mt",
            "mr",
            "mb",
            "ml",
            "mx",
            "my",
            "p",
            "pt",
            "pr",
            "pb",
            "pl",
            "px",
            "py",
            "width",
            "w",
            "fontSize",
            "f",
            "color",
            "bg",
            "fontFamily",
            "font",
            "textAlign",
            "align",
            "lineHeight",
            "fontWeight",
            "letterSpacing",
            "display",
            "maxWidth",
            "minWidth",
            "height",
            "maxHeight",
            "minHeight",
            "size",
            "ratio",
            "alignItems",
            "alignContent",
            "justifyContent",
            "justify",
            "flexWrap",
            "wrap",
            "flexBasis",
            "flexDirection",
            "flex",
            "justifySelf",
            "alignSelf",
            "order",
            "gridGap",
            "gridColumnGap",
            "gridRowGap",
            "gridColumn",
            "gridRow",
            "gridAutoFlow",
            "gridAutoColumns",
            "gridAutoRows",
            "gridTemplateColumns",
            "gridTemplateRows",
            "border",
            "borderTop",
            "borderRight",
            "borderBottom",
            "borderLeft",
            "borderColor",
            "borderRadius",
            "boxShadow",
            "background",
            "backgroundImage",
            "bgImage",
            "backgroundSize",
            "bgSize",
            "backgroundPosition",
            "bgPosition",
            "backgroundRepeat",
            "bgRepeat",
            "position",
            "zIndex",
            "top",
            "right",
            "bottom",
            "left",
            "hover",
            "focus",
            "active",
            "disabledStyle",
            "textStyle",
            "colors",
            "buttonStyle",
            "borderWidth",
          ]
        }
        className="c0"
        is="div"
      >
        <div
          className="c0"
        >
          <Radio__RadioInput
            disabled={true}
            type="radio"
          >
            <input
              className="c1"
              disabled={true}
              type="radio"
            />
          </Radio__RadioInput>
          <Radio__RadioIcon>
            <Clean.div
              className="c2"
            >
              <Clean.div
                blacklist={
                  Array [
                    "m",
                    "mt",
                    "mr",
                    "mb",
                    "ml",
                    "mx",
                    "my",
                    "p",
                    "pt",
                    "pr",
                    "pb",
                    "pl",
                    "px",
                    "py",
                    "width",
                    "w",
                    "fontSize",
                    "f",
                    "color",
                    "bg",
                    "fontFamily",
                    "font",
                    "textAlign",
                    "align",
                    "lineHeight",
                    "fontWeight",
                    "letterSpacing",
                    "display",
                    "maxWidth",
                    "minWidth",
                    "height",
                    "maxHeight",
                    "minHeight",
                    "size",
                    "ratio",
                    "alignItems",
                    "alignContent",
                    "justifyContent",
                    "justify",
                    "flexWrap",
                    "wrap",
                    "flexBasis",
                    "flexDirection",
                    "flex",
                    "justifySelf",
                    "alignSelf",
                    "order",
                    "gridGap",
                    "gridColumnGap",
                    "gridRowGap",
                    "gridColumn",
                    "gridRow",
                    "gridAutoFlow",
                    "gridAutoColumns",
                    "gridAutoRows",
                    "gridTemplateColumns",
                    "gridTemplateRows",
                    "border",
                    "borderTop",
                    "borderRight",
                    "borderBottom",
                    "borderLeft",
                    "borderColor",
                    "borderRadius",
                    "boxShadow",
                    "background",
                    "backgroundImage",
                    "bgImage",
                    "backgroundSize",
                    "bgSize",
                    "backgroundPosition",
                    "bgPosition",
                    "backgroundRepeat",
                    "bgRepeat",
                    "position",
                    "zIndex",
                    "top",
                    "right",
                    "bottom",
                    "left",
                    "hover",
                    "focus",
                    "active",
                    "disabledStyle",
                    "textStyle",
                    "colors",
                    "buttonStyle",
                    "borderWidth",
                  ]
                }
                className="c2"
                is="div"
              >
                <div
                  className="c2"
                />
              </Clean.div>
            </Clean.div>
          </Radio__RadioIcon>
        </div>
      </Clean.div>
    </Clean.div>
  </Radio__RadioWrapper>
</Radio>
`;
